vpc/service/gc_v3.go

```diff
@@ -170,6 +170,7 @@ FROM branch_eni_attachments
 JOIN assignments ON branch_eni_attachments.association_id = assignments.branch_eni_association
 WHERE trunk_eni = $1
 AND assignments.assignment_id NOT IN (SELECT unnest($2::text[]))
+AND created_at < now() - INTERVAL '5 minutes'
```

Comment:
It would be nice if this was a const at the top of the file, but I'm not too fussed.

Reply:
I couldn't figure out how to do this.

Reply:
PRs welcome. :P. (No, seriously, https://github.com/lib/pq/issues/78)